addons/mail/static/src/variables.scss

```diff
@@ -2,13 +2,33 @@
 // Variables
 // -----------------------------------------------------------------------------
 
-$o-mail-thread-window-zindex: $zindex-modal + 1 !default;
-
+$o-mail-attachment-image-size: 100px !default;
+/**
+ * Generic border-radius used in the context of messaging, with some size variations
```

Comment:
dot at the end.
Also specify that it is currently needed because the generic `$border-radius` variations have all been set to 0 in enterprise.